examples/pytorch/metapath2vec/main2.py

```diff
@@ -0,0 +1,212 @@
+import torch
```

Comment:
You need to move this module to `dgl/python/dgl/nn/pytorch/`. You can create a file `network_emb.py` there.